src/components/common/CreateBaleModal.js

```diff
@@ -27,27 +27,17 @@ class CreateBaleModal extends Component {
 
   getMaterials() {
     const pickerMaterial = [];
-    pickerMaterial.push(
-      <Picker.Item
-        key={999}
-        label={strings.selectMaterial}
-        value={false}
-      />,
-    );
+    pickerMaterial.push(<Picker.Item key={999} label={strings.selectMaterial} value={false} />);
     this.materials.map((material) => {
-      pickerMaterial.push(<Picker.Item
-        key={material.id}
-        label={material.name}
-        value={material.name}
-      />);
+      pickerMaterial.push(<Picker.Item key={material.id} label={material.name} value={material.value} />);
```

Comment:
Watch out for line length, it must not exceed 100 characters, thats why it was the way it was. Line 30 was also indented for consistency.